internal/changestream/eventmultiplexer/subscription.go

```diff
@@ -34,20 +35,24 @@ type requestSubscriptionResult struct {
 // subscription represents a subscriber in the event queue. It holds a tomb, so
 // that we can tie the lifecycle of a subscription to the event queue.
 type subscription struct {
-	tomb tomb.Tomb
-	id   uint64
+	tomb      tomb.Tomb
+	id        uint64
+	unsubOnce sync.Once
 
-	topics        map[string]struct{}
-	changes       chan ChangeSet
-	unsubscribeFn func()
+	topics  map[string]struct{}
+	changes chan ChangeSet
+
+	dispatchTimout time.Duration
+	unsubscribeFn  func()
 }
 
 func newSubscription(id uint64, unsubscribeFn func()) *subscription {
 	sub := &subscription{
-		id:            id,
-		changes:       make(chan ChangeSet),
-		topics:        make(map[string]struct{}),
-		unsubscribeFn: unsubscribeFn,
+		id:             id,
+		changes:        make(chan ChangeSet),
+		topics:         make(map[string]struct{}),
+		dispatchTimout: DefaultSignalTimeout,
```

Comment:
Typo:
```diff
-		dispatchTimout: DefaultSignalTimeout,
+		dispatchTimeout: DefaultSignalTimeout,
```

Reply:
Which is here: https://github.com/juju/juju/pull/20021.